app/Resources/AppBundle/assets/js/main.js

```diff
@@ -2,11 +2,11 @@ require('./MovingCards');
 
 // materialize-css element functions
 $(document).ready(function(){
+    $(".dropdown-button").dropdown();
```

Comment:
We could argue about the granularity.  We could call it materialize.js too and put the activator for collectibles in there too.